packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -682,7 +696,7 @@ export class ListItem
               aria-setsize={setSize}
               class={{
                 [CSS.container]: true,
-                [CSS.containerHover]: true,
+                [CSS.containerHover]: !containerNotInteractive || containerInteractive,
```

Comment:
This naming a bit misleading. Since `containerInteractive` and `containerNotInteractive` are opposites, this ends up reading as `containerInteractive || containerInteractive`.